v23.1 update from main.

rderbier opened the pull request:
<!--
Title: Please use the following format for your PR title:  topic(area): details
- The "topic" should be one of the following: Docs, Nav or Chore
- The "area" is the feature (i.e., "GraphQL"), area of the docs (i.e., "Deployment"), or "Other" (for typo fixes and other bug-fix PRs). 
Sample Titles:
  Docs(GraphQL): Document the @deprecated annotation
  Chore(Other): cherry-pick updates from main to release/v20.11

Description: Please include the following in your PR description:
1. A brief, clear description of the change.
2. Link to any related PRs or discuss.dgraph.io posts.
3. If this PR corresponds to a JIRA issue, include "Fixes DOC-###" in the PR description.
3. If this is a breaking change, please put "[Breaking]" in the title. In the description, please put a note with exactly who these changes are breaking for.
4. If you are creating a PR in `main` and you know it needs to be cherry-picked to a release branch, please mention that in your PR description (for example: "cherry-pick to v20.07"). Cherry-pick PRs should reference the original PR.

Note: Create and edit docs in the `main` branch when you can, so that we only cherry-pick out of `main`, not into `main`.
-->

Jonathan-Zollinger committed:
Docs(import data): fix small typo (#641)

## Description
This is just a simple typo I found.

rderbier committed:
update graphql subscription doc (#645)

add example using URQL and subscriptions-transport-ws

rderbier committed:
add Clerk.com config example (#646)

Add an example of Dgraph.Authorization configuration with Clerk.com auth
provider.

shivaji-kharse committed:
docs(dql):Add docs for unique directive (#644)

shivaji-kharse committed:
Docs(GraphQL):add docs for restoreTenant API (#647)

rderbier committed:
Update RBAC-rules.md (#648)

Clarify that a claim can be an array of strings in RBAC rules.

ryanfoxtyler committed:
Update release table (#652)

Updating the release table in docs to:
1. include the latest release (v23.1)
2. make the table more maintainable/default current
3. omit patch releases

ryanfoxtyler committed:
Update build.sh

Removing auto-version selection, as docs must be ready

ryanfoxtyler committed:
Update build.sh (#654)

Undo change

ryanfoxtyler committed:
Update CODEOWNERS (#656)

Updating codeowners to map to GitHub Team for easier management

rderbier committed:
Update README.md

ryanfoxtyler committed:
fix Dgraph Tour link (#665)

ryanfoxtyler committed:
Create stale.yml

Commit:
updated badger metrics (#632)

<!--
Title: Please use the following format for your PR title: topic(area):
details
- The "topic" should be one of the following: Docs, Nav or Chore
- The "area" is the feature (i.e., "GraphQL"), area of the docs (i.e.,
"Deployment"), or "Other" (for typo fixes and other bug-fix PRs).
Sample Titles:
  Docs(GraphQL): Document the @deprecated annotation
  Chore(Other): cherry-pick updates from main to release/v20.11

Description: Please include the following in your PR description:
1. A brief, clear description of the change.
2. Link to any related PRs or discuss.dgraph.io posts.
3. If this PR corresponds to a JIRA issue, include "Fixes DOC-###" in
the PR description.
3. If this is a breaking change, please put "[Breaking]" in the title.
In the description, please put a note with exactly who these changes are
breaking for.
4. If you are creating a PR in `main` and you know it needs to be
cherry-picked to a release branch, please mention that in your PR
description (for example: "cherry-pick to v20.07"). Cherry-pick PRs
should reference the original PR.

Note: Create and edit docs in the `main` branch when you can, so that we
only cherry-pick out of `main`, not into `main`.
-->

Co-authored-by: Harshil Goel <harshil@dgraph.io>

EronWright committed:
Docs(GraphQL): Fix typo "directory" vs "directive" (#671)

<!--
Title: Please use the following format for your PR title: topic(area):
details
- The "topic" should be one of the following: Docs, Nav or Chore
- The "area" is the feature (i.e., "GraphQL"), area of the docs (i.e.,
"Deployment"), or "Other" (for typo fixes and other bug-fix PRs).
Sample Titles:
  Docs(GraphQL): Document the @deprecated annotation
  Chore(Other): cherry-pick updates from main to release/v20.11

Description: Please include the following in your PR description:
1. A brief, clear description of the change.
2. Link to any related PRs or discuss.dgraph.io posts.
3. If this PR corresponds to a JIRA issue, include "Fixes DOC-###" in
the PR description.
3. If this is a breaking change, please put "[Breaking]" in the title.
In the description, please put a note with exactly who these changes are
breaking for.
4. If you are creating a PR in `main` and you know it needs to be
cherry-picked to a release branch, please mention that in your PR
description (for example: "cherry-pick to v20.07"). Cherry-pick PRs
should reference the original PR.

Note: Create and edit docs in the `main` branch when you can, so that we
only cherry-pick out of `main`, not into `main`.
-->

Fixes a minor typo in the documentation.

Co-authored-by: Ryan Fox-Tyler <60440289+ryanfoxtyler@users.noreply.github.com>

dependabot[bot] committed:
Bump tj-actions/changed-files from 35 to 41 in /.github/workflows (#672)

Bumps
[tj-actions/changed-files](https://github.com/tj-actions/changed-files)
from 35 to 41.
<details>
<summary>Release notes</summary>
<p><em>Sourced from <a
href="https://github.com/tj-actions/changed-files/releases">tj-actions/changed-files's
releases</a>.</em></p>
<blockquote>
<h2>v41</h2>
<h1>Changes in v41.1.2</h1>
<h2>What's Changed</h2>
<ul>
<li>Upgraded to v41.1.1 by <a
href="https://github.com/tj-actions-bot"><code>@​tj-actions-bot</code></a>
in <a
href="https://redirect.github.com/tj-actions/changed-files/pull/1854">tj-actions/changed-files#1854</a></li>
<li>chore(deps): update dependency prettier to v3.2.2 by <a
href="https://github.com/renovate"><code>@​renovate</code></a> in <a
href="https://redirect.github.com/tj-actions/changed-files/pull/1855">tj-actions/changed-files#1855</a></li>
<li>chore(deps): lock file maintenance by <a
href="https://github.com/renovate"><code>@​renovate</code></a> in <a
href="https://redirect.github.com/tj-actions/changed-files/pull/1856">tj-actions/changed-files#1856</a></li>
<li>chore(deps): update dependency <code>@​types/node</code> to v20.11.1
by <a href="https://github.com/renovate"><code>@​renovate</code></a> in
<a
href="https://redirect.github.com/tj-actions/changed-files/pull/1857">tj-actions/changed-files#1857</a></li>
<li>chore(deps): update dependency <code>@​types/node</code> to v20.11.2
by <a href="https://github.com/renovate"><code>@​renovate</code></a> in
<a
href="https://redirect.github.com/tj-actions/changed-files/pull/1858">tj-actions/changed-files#1858</a></li>
<li>chore(deps): update typescript-eslint monorepo to v6.19.0 by <a
href="https://github.com/renovate"><code>@​renovate</code></a> in <a
href="https://redirect.github.com/tj-actions/changed-files/pull/1860">tj-actions/changed-files#1860</a></li>
<li>feat: enhance error handling and working directory resolution by <a
href="https://github.com/jackton1"><code>@​jackton1</code></a> in <a
href="https://redirect.github.com/tj-actions/changed-files/pull/1859">tj-actions/changed-files#1859</a></li>
<li>chore(deps): update dependency <code>@​types/node</code> to v20.11.3
by <a href="https://github.com/renovate"><code>@​renovate</code></a> in
<a
href="https://redirect.github.com/tj-actions/changed-files/pull/1862">tj-actions/changed-files#1862</a></li>
<li>chore(deps): update dependency <code>@​types/node</code> to v20.11.4
by <a href="https://github.com/renovate"><code>@​renovate</code></a> in
<a
href="https://redirect.github.com/tj-actions/changed-files/pull/1863">tj-actions/changed-files#1863</a></li>
<li>chore(deps): update tj-actions/eslint-changed-files action to v22 by
<a href="https://github.com/renovate"><code>@​renovate</code></a> in <a
href="https://redirect.github.com/tj-actions/changed-files/pull/1864">tj-actions/changed-files#1864</a></li>
<li>chore(deps): update dependency prettier to v3.2.3 by <a
href="https://github.com/renovate"><code>@​renovate</code></a> in <a
href="https://redirect.github.com/tj-actions/changed-files/pull/1867">tj-actions/changed-files#1867</a></li>
<li>fix: bug with incorrect action path by <a
href="https://github.com/jackton1"><code>@​jackton1</code></a> in <a
href="https://redirect.github.com/tj-actions/changed-files/pull/1866">tj-actions/changed-files#1866</a></li>
<li>chore(deps): update dependency <code>@​types/node</code> to v20.11.5
by <a href="https://github.com/renovate"><code>@​renovate</code></a> in
<a
href="https://redirect.github.com/tj-actions/changed-files/pull/1868">tj-actions/changed-files#1868</a></li>
</ul>
<p><strong>Full Changelog</strong>: <a
href="https://github.com/tj-actions/changed-files/compare/v41...v41.1.2">https://github.com/tj-actions/changed-files/compare/v41...v41.1.2</a></p>
<hr />
<h1>Changes in v41.1.1</h1>
<h2>What's Changed</h2>
<ul>
<li>Upgraded to v41.1.0 by <a
href="https://github.com/tj-actions-bot"><code>@​tj-actions-bot</code></a>
in <a
href="https://redirect.github.com/tj-actions/changed-files/pull/1852">tj-actions/changed-files#1852</a></li>
<li>fix: bug with inaccurate warnings by <a
href="https://github.com/jackton1"><code>@​jackton1</code></a> in <a
href="https://redirect.github.com/tj-actions/changed-files/pull/1853">tj-actions/changed-files#1853</a></li>
</ul>
<p><strong>Full Changelog</strong>: <a
href="https://github.com/tj-actions/changed-files/compare/v41...v41.1.1">https://github.com/tj-actions/changed-files/compare/v41...v41.1.1</a></p>
<hr />
<h1>Changes in v41.1.0</h1>
<h2>What's Changed</h2>
<ul>
<li>Upgraded to v41.0.1 by <a
href="https://github.com/tj-actions-bot"><code>@​tj-actions-bot</code></a>
in <a
href="https://redirect.github.com/tj-actions/changed-files/pull/1816">tj-actions/changed-files#1816</a></li>
<li>chore: update matrix-test.yml by <a
href="https://github.com/jackton1"><code>@​jackton1</code></a> in <a
href="https://redirect.github.com/tj-actions/changed-files/pull/1814">tj-actions/changed-files#1814</a></li>
<li>chore(deps): update typescript-eslint monorepo to v6.16.0 by <a
href="https://github.com/renovate"><code>@​renovate</code></a> in <a
href="https://redirect.github.com/tj-actions/changed-files/pull/1819">tj-actions/changed-files#1819</a></li>
<li>chore(deps): update tj-actions/verify-changed-files action to v17 by
<a href="https://github.com/renovate"><code>@​renovate</code></a> in <a
href="https://redirect.github.com/tj-actions/changed-files/pull/1820">tj-actions/changed-files#1820</a></li>
<li>chore(deps): update dependency <code>@​types/node</code> to v20.10.6
by <a href="https://github.com/renovate"><code>@​renovate</code></a> in
<a
href="https://redirect.github.com/tj-actions/changed-files/pull/1821">tj-actions/changed-files#1821</a></li>
<li>chore(deps): lock file maintenance by <a
href="https://github.com/renovate"><code>@​renovate</code></a> in <a
href="https://redirect.github.com/tj-actions/changed-files/pull/1822">tj-actions/changed-files#1822</a></li>
<li>chore(deps): update typescript-eslint monorepo to v6.17.0 by <a
href="https://github.com/renovate"><code>@​renovate</code></a> in <a
href="https://redirect.github.com/tj-actions/changed-files/pull/1823">tj-actions/changed-files#1823</a></li>
<li>chore(deps): update dependency eslint-plugin-jest to v27.6.1 by <a
href="https://github.com/renovate"><code>@​renovate</code></a> in <a
href="https://redirect.github.com/tj-actions/changed-files/pull/1824">tj-actions/changed-files#1824</a></li>
<li>chore(deps): update typescript-eslint monorepo to v6.18.0 by <a
href="https://github.com/renovate"><code>@​renovate</code></a> in <a
href="https://redirect.github.com/tj-actions/changed-files/pull/1827">tj-actions/changed-files#1827</a></li>
<li>chore: create workflow-run-test.yml by <a
href="https://github.com/jackton1"><code>@​jackton1</code></a> in <a
href="https://redirect.github.com/tj-actions/changed-files/pull/1828">tj-actions/changed-files#1828</a></li>
<li>Updated README.md by <a
href="https://github.com/tj-actions-bot"><code>@​tj-actions-bot</code></a>
in <a
href="https://redirect.github.com/tj-actions/changed-files/pull/1830">tj-actions/changed-files#1830</a></li>
<li>chore(deps): update dependency <code>@​types/node</code> to v20.10.7
by <a href="https://github.com/renovate"><code>@​renovate</code></a> in
<a
href="https://redirect.github.com/tj-actions/changed-files/pull/1831">tj-actions/changed-files#1831</a></li>
<li>chore(deps): lock file maintenance by <a
href="https://github.com/renovate"><code>@​renovate</code></a> in <a
href="https://redirect.github.com/tj-actions/changed-files/pull/1833">tj-actions/changed-files#1833</a></li>
<li>chore: update jest.config.js by <a
href="https://github.com/jackton1"><code>@​jackton1</code></a> in <a
href="https://redirect.github.com/tj-actions/changed-files/pull/1834">tj-actions/changed-files#1834</a></li>
<li>chore(deps): update typescript-eslint monorepo to v6.18.1 by <a
href="https://github.com/renovate"><code>@​renovate</code></a> in <a
href="https://redirect.github.com/tj-actions/changed-files/pull/1835">tj-actions/changed-files#1835</a></li>
<li>chore(deps-dev): bump <code>@​types/node</code> from 20.10.7 to
20.10.8 by <a
href="https://github.com/dependabot"><code>@​dependabot</code></a> in <a
href="https://redirect.github.com/tj-actions/changed-files/pull/1836">tj-actions/changed-files#1836</a></li>
</ul>
<!-- raw HTML omitted -->
</blockquote>
<p>... (truncated)</p>
</details>
<details>
<summary>Changelog</summary>
<p><em>Sourced from <a
href="https://github.com/tj-actions/changed-files/blob/main/HISTORY.md">tj-actions/changed-files's
changelog</a>.</em></p>
<blockquote>
<h1>Changelog</h1>
<h1><a
href="https://github.com/tj-actions/changed-files/compare/v44.5.4...v44.5.5">44.5.5</a>
- (2024-06-24)</h1>
<h2><!-- raw HTML omitted -->⚙️ Miscellaneous Tasks</h2>
<ul>
<li>Remove debug lines (<a
href="https://redirect.github.com/tj-actions/changed-files/issues/2166">#2166</a>)
(<a
href="https://github.com/tj-actions/changed-files/commit/cc733854b1f224978ef800d29e4709d5ee2883e4">cc73385</a>)
- (Tonye Jack)</li>
</ul>
<h2><!-- raw HTML omitted -->⬆️ Upgrades</h2>
<ul>
<li>Upgraded to v44.5.4 (<a
href="https://redirect.github.com/tj-actions/changed-files/issues/2165">#2165</a>)</li>
</ul>
<p>Co-authored-by: jackton1 <a
href="mailto:17484350+jackton1@users.noreply.github.com">17484350+jackton1@users.noreply.github.com</a>
(<a
href="https://github.com/tj-actions/changed-files/commit/0b99ecfd4efe684550ba7ef222cdf6b36964c890">0b99ecf</a>)
- (tj-actions[bot])</p>
<h1><a
href="https://github.com/tj-actions/changed-files/compare/v44.5.3...v44.5.4">44.5.4</a>
- (2024-06-23)</h1>
<h2><!-- raw HTML omitted -->🐛 Bug Fixes</h2>
<ul>
<li>Error using since_last_remote_commit with the first PR commit (<a
href="https://redirect.github.com/tj-actions/changed-files/issues/2163">#2163</a>)
(<a
href="https://github.com/tj-actions/changed-files/commit/cc3bbb0c526f8ee1d282f8c5f9f4e50745a5b457">cc3bbb0</a>)
- (Tonye Jack)</li>
<li><strong>deps:</strong> Update dependency <code>@​octokit/rest</code>
to v21 (<a
href="https://redirect.github.com/tj-actions/changed-files/issues/2158">#2158</a>)
(<a
href="https://github.com/tj-actions/changed-files/commit/7891350b8e5530e6284113b15f753f9673276d1c">7891350</a>)
- (renovate[bot])</li>
</ul>
<h2><!-- raw HTML omitted -->⚙️ Miscellaneous Tasks</h2>
<ul>
<li><strong>deps:</strong> Update dependency <code>@​types/node</code>
to v20.14.8 (<a
href="https://github.com/tj-actions/changed-files/commit/373db58d9d98a3002e86f6881421378d7be4de26">373db58</a>)
- (renovate[bot])</li>
<li><strong>deps-dev:</strong> Bump <code>@​types/uuid</code> from 9.0.8
to 10.0.0 (<a
href="https://redirect.github.com/tj-actions/changed-files/issues/2160">#2160</a>)
(<a
href="https://github.com/tj-actions/changed-files/commit/9161a8dcde05a2a3b845745c47574885a33fd5e5">9161a8d</a>)
- (dependabot[bot])</li>
<li><strong>deps:</strong> Update dependency <code>@​types/node</code>
to v20.14.7 (<a
href="https://github.com/tj-actions/changed-files/commit/457e041909e8e6db7872dbc85f43d3fea4a72f72">457e041</a>)
- (renovate[bot])</li>
</ul>
<h2><!-- raw HTML omitted -->⬆️ Upgrades</h2>
<ul>
<li>Upgraded to v44.5.3 (<a
href="https://redirect.github.com/tj-actions/changed-files/issues/2154">#2154</a>)</li>
</ul>
<p>Co-authored-by: jackton1 <a
href="mailto:17484350+jackton1@users.noreply.github.com">17484350+jackton1@users.noreply.github.com</a>
(<a
href="https://github.com/tj-actions/changed-files/commit/e13fb92d451df65cea1041a85dd4a3b24153d155">e13fb92</a>)
- (tj-actions[bot])</p>
<h1><a
href="https://github.com/tj-actions/changed-files/compare/v44.5.2...v44.5.3">44.5.3</a>
- (2024-06-19)</h1>
<h2><!-- raw HTML omitted -->🐛 Bug Fixes</h2>
<ul>
<li><strong>deps:</strong> Update dependency yaml to v2.4.5 (<a
href="https://github.com/tj-actions/changed-files/commit/57a81a39f3c336e860a2312a2487a8a2e6d495ce">57a81a3</a>)
- (renovate[bot])</li>
<li><strong>deps:</strong> Update dependency yaml to v2.4.4 (<a
href="https://github.com/tj-actions/changed-files/commit/3de07763cf5f7eb991d59ca7894be8df0f36c80c">3de0776</a>)
- (renovate[bot])</li>
<li><strong>deps:</strong> Update dependency yaml to v2.4.3 (<a
href="https://redirect.github.com/tj-actions/changed-files/issues/2120">#2120</a>)
(<a
href="https://github.com/tj-actions/changed-files/commit/60268df189b72237cfcbe6ec2937367ef2b2b75d">60268df</a>)
- (renovate[bot])</li>
</ul>
<h2><!-- raw HTML omitted -->➕ Add</h2>
<ul>
<li>Added missing changes and modified dist assets.
(<a
href="https://github.com/tj-actions/changed-files/commit/2972fe4d6b07bd6de22186555c2046a94a3aa302">2972fe4</a>)
- (GitHub Action)</li>
<li>Added missing changes and modified dist assets.
(<a
href="https://github.com/tj-actions/changed-files/commit/1d8732f877032fb30b668e90ed6e4256ce87e263">1d8732f</a>)
- (GitHub Action)</li>
</ul>
<h2><!-- raw HTML omitted -->⚙️ Miscellaneous Tasks</h2>
<!-- raw HTML omitted -->
</blockquote>
<p>... (truncated)</p>
</details>
<details>
<summary>Commits</summary>
<ul>
<li><a
href="https://github.com/tj-actions/changed-files/commit/cbda684547adc8c052d50711417fa61b428a9f88"><code>cbda684</code></a>
chore(deps): update dependency <code>@​types/node</code> to
v20.11.5</li>
<li><a
href="https://github.com/tj-actions/changed-files/commit/3f8189989ba6fd9b3b9593ffe650236b3f1fcb55"><code>3f81899</code></a>
fix: bug with incorrect action path (<a
href="https://redirect.github.com/tj-actions/changed-files/issues/1866">#1866</a>)</li>
<li><a
href="https://github.com/tj-actions/changed-files/commit/67a1f54f6f5ec7ee87c57eb7876a7d6dfdcc59a1"><code>67a1f54</code></a>
chore(deps): update dependency prettier to v3.2.3</li>
<li><a
href="https://github.com/tj-actions/changed-files/commit/99248a443855d73284abf52520f897dba851b914"><code>99248a4</code></a>
chore(deps): update tj-actions/eslint-changed-files action to v22 (<a
href="https://redirect.github.com/tj-actions/changed-files/issues/1864">#1864</a>)</li>
<li><a
href="https://github.com/tj-actions/changed-files/commit/878743189ba0ba42d467a736923b07102f0e348c"><code>8787431</code></a>
chore(deps): update dependency <code>@​types/node</code> to
v20.11.4</li>
<li><a
href="https://github.com/tj-actions/changed-files/commit/98d1d84e2f7a404c425df4e44dceb74a03920ac8"><code>98d1d84</code></a>
chore(deps): update dependency <code>@​types/node</code> to
v20.11.3</li>
<li><a
href="https://github.com/tj-actions/changed-files/commit/a60bf3759e069549b60c8da1284ec83e0398a1a4"><code>a60bf37</code></a>
feat: enhance error handling and working directory resolution (<a
href="https://redirect.github.com/tj-actions/changed-files/issues/1859">#1859</a>)</li>
<li><a
href="https://github.com/tj-actions/changed-files/commit/bc46e4c4222c3926a70378d183f0b387d3a9e9a8"><code>bc46e4c</code></a>
chore(deps): update typescript-eslint monorepo to v6.19.0</li>
<li><a
href="https://github.com/tj-actions/changed-files/commit/fba40673489d49e860c15a444c134d887ead1f3a"><code>fba4067</code></a>
chore(deps): update dependency <code>@​types/node</code> to
v20.11.2</li>
<li><a
href="https://github.com/tj-actions/changed-files/commit/e4b86747326bc58eb230d62188ebdd66b73721a9"><code>e4b8674</code></a>
chore(deps): update dependency <code>@​types/node</code> to
v20.11.1</li>
<li>Additional commits viewable in <a
href="https://github.com/tj-actions/changed-files/compare/v35...v41">compare
view</a></li>
</ul>
</details>
<br />


[![Dependabot compatibility
score](https://dependabot-badges.githubapp.com/badges/compatibility_score?dependency-name=tj-actions/changed-files&package-manager=github_actions&previous-version=35&new-version=41)](https://docs.github.com/en/github/managing-security-vulnerabilities/about-dependabot-security-updates#about-compatibility-scores)

Dependabot will resolve any conflicts with this PR as long as you don't
alter it yourself. You can also trigger a rebase manually by commenting
`@dependabot rebase`.

[//]: # (dependabot-automerge-start)
[//]: # (dependabot-automerge-end)

---

<details>
<summary>Dependabot commands and options</summary>
<br />

You can trigger Dependabot actions by commenting on this PR:
- `@dependabot rebase` will rebase this PR
- `@dependabot recreate` will recreate this PR, overwriting any edits
that have been made to it
- `@dependabot merge` will merge this PR after your CI passes on it
- `@dependabot squash and merge` will squash and merge this PR after
your CI passes on it
- `@dependabot cancel merge` will cancel a previously requested merge
and block automerging
- `@dependabot reopen` will reopen this PR if it is closed
- `@dependabot close` will close this PR and stop Dependabot recreating
it. You can achieve the same result by closing it manually
- `@dependabot show <dependency name> ignore conditions` will show all
of the ignore conditions of the specified dependency
- `@dependabot ignore this major version` will close this PR and stop
Dependabot creating any more for this major version (unless you reopen
the PR or upgrade to it yourself)
- `@dependabot ignore this minor version` will close this PR and stop
Dependabot creating any more for this minor version (unless you reopen
the PR or upgrade to it yourself)
- `@dependabot ignore this dependency` will close this PR and stop
Dependabot creating any more for this dependency (unless you reopen the
PR or upgrade to it yourself)
You can disable automated security fix PRs for this repo from the
[Security Alerts
page](https://github.com/dgraph-io/dgraph-docs/network/alerts).

</details>

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/dgraph-io/dgraph-docs?pullRequest=660) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.io/dgraph-io/dgraph-docs?pullRequest=660) before we can accept your contribution.<br/>**6** out of **7** committers have signed the CLA.<br/><br/>:white_check_mark: Jonathan-Zollinger<br/>:white_check_mark: ryanfoxtyler<br/>:white_check_mark: shivaji-dgraph<br/>:white_check_mark: EronWright<br/>:white_check_mark: harshil-goel<br/>:white_check_mark: rderbier<br/>:x: dependabot[bot]<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/dgraph-io/dgraph-docs?pullRequest=660) it.</sub>

kevinmingtarja committed:
Docs(Enterprise): Notify that offline restore feature is deprecated (#318)

Description:

The offline restore feature is deprecated (v21.03.0 was the last version
to have it), but it is not stated in the docs, which might confuse
readers. So, I think we should do something to notify of this change.

ryanfoxtyler committed:
Update stale.yml

alexatcanva committed:
docs(dql): Updated schema alter from date, to datetime (#657)

<!--
Title: Please use the following format for your PR title: topic(area):
details
- The "topic" should be one of the following: Docs, Nav or Chore
- The "area" is the feature (i.e., "GraphQL"), area of the docs (i.e.,
"Deployment"), or "Other" (for typo fixes and other bug-fix PRs).
Sample Titles:
  Docs(GraphQL): Document the @deprecated annotation
  Chore(Other): cherry-pick updates from main to release/v20.11

Description: Please include the following in your PR description:
1. A brief, clear description of the change.
2. Link to any related PRs or discuss.dgraph.io posts.
3. If this PR corresponds to a JIRA issue, include "Fixes DOC-###" in
the PR description.
3. If this is a breaking change, please put "[Breaking]" in the title.
In the description, please put a note with exactly who these changes are
breaking for.
4. If you are creating a PR in `main` and you know it needs to be
cherry-picked to a release branch, please mention that in your PR
description (for example: "cherry-pick to v20.07"). Cherry-pick PRs
should reference the original PR.

Note: Create and edit docs in the `main` branch when you can, so that we
only cherry-pick out of `main`, not into `main`.
-->

## Description

While following the
https://dgraph.io/docs/dql/dql-get-started#step-4-alter-schema
documentation I noticed that the described type for `release_date` was
of type `date` not `datetime`.

It appears that in later versions of the Ratel UI (or perhaps it's
always been?) `date` is now `datetime`.

This PR modifies the documentation from `date` to `datetime`.


![image](https://github.com/dgraph-io/dgraph-docs/assets/86993591/656aea30-ece1-4514-be0f-543ccb067c5b)

## Notes

This is my first contribution to the dgraph docs repo, please let me
know if there's anything else I can do to make this a better PR!

Many thanks!

Co-authored-by: Ryan Fox-Tyler <60440289+ryanfoxtyler@users.noreply.github.com>

Rajakavitha1 committed:
[Docs] Dgraph CLI Reference (#630)

Remove references to raft migrate in the CLI.
Fixes: https://github.com/dgraph-io/dgraph-docs/issues/600

Description: Please include the following in your PR description:
1. A brief, clear description of the change.
2. Link to any related PRs or discuss.dgraph.io posts.
3. If this PR corresponds to a JIRA issue, include "Fixes DOC-###" in
the PR description.
3. If this is a breaking change, please put "[Breaking]" in the title.
In the description, please put a note with exactly who these changes are
breaking for.
4. If you are creating a PR in `main` and you know it needs to be
cherry-picked to a release branch, please mention that in your PR
description (for example: "cherry-pick to v20.07"). Cherry-pick PRs
should reference the original PR.

Note: Create and edit docs in the `main` branch when you can, so that we
only cherry-pick out of `main`, not into `main`.
-->

rderbier commented:
In order to clean old PR I have manually pushed all modifications to releases/v23.1
Closing this PR.
